CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- `tt replicaset expel`:  command to expel an instance from the tarantool replicaset with
+ cluster config (3.0) or cartridge orchestrator.
+- `tt replicaset vshard`: module to manage vshard in the tarantool replicaset.
+  * `tt replicaset vshard bootstrap`: command to bootstrap vshard.
+- `tt cluster replicaset expel`: command to expel an instance from the replicaset via cluster
+ config storage.
```

Comment:
Please, make a separate commit with the move of `expel` changelog entries. It will be easier to rebase/we could cherry-pick the commit in another pull request.

Reply:
Done. Thank you!